src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        ColorSupplier suppler = new ColorSupplier();
+        int ballNumber = random.nextInt(100);
+        String ballColor = suppler.getRandomColor();
+        return new Ball(ballColor,ballNumber);
+    }
+}
```

Comment:
The `Lottery` class is creating new instances of `Random` and `ColorSupplier` every time the `getRandomBall` method is called. This is not efficient as you can reuse the same `Random` and `ColorSupplier` instances. Consider declaring them as private final fields in the `Lottery` class and initialize them in the constructor or directly in the field declaration.